ipamd/ipamd.go

```diff
@@ -305,17 +313,9 @@ func (c *IPAMContext) nodeInit() error {
 	for _, eni := range enis {
 		log.Debugf("Discovered ENI %s, trying to set it up", eni.ENIID)
 		// Retry ENI sync
-		retry := 0
-		for {
-			retry++
-			err = c.setupENI(eni.ENIID, eni)
-			if retry > maxRetryCheckENI {
-				log.Errorf("Unable to discover attached IPs for ENI from metadata service")
-				ipamdErrInc("waitENIAttachedMaxRetryExceeded")
-				break
-			}
-
-			if err != nil {
+		setup := false
+		for retry := 0; retry <= maxRetryCheckENI; retry++ {
```

Comment:
again, try to keep style-only changes like this in a separate PR. this change isn't critical to the new functionality being added and while I prefer your code over the original style, it's tougher for reviewers to review the functional changes added in this PR from the style-only changes you've made.